test/k8sT/Services.go

```diff
@@ -120,6 +120,8 @@ var _ = Describe("K8sServicesTest", func() {
 		BeforeEach(func() {
 			res := kubectl.Apply(demoYAML)
 			res.ExpectSuccess("unable to apply %s", demoYAML)
+			ExpectDaemonSetReady(kubectl, helpers.DefaultNamespace, "testds", helpers.HelperTimeout)
```

Comment:
There is no `DaemonSet` called `testds` in this test - perhaps you  meant `ExpectDeployReady`, as `demo.yaml` contains deployments?